user_guides/jstl/src/main/jbake/content/attributes.conf

```diff
@@ -1,12 +1,12 @@
 :TechnologyFullName: Jakarta Standard Tag Library
 :TechnologyShortName: Tag Library
 :LegacyAcronym: JSTL		   
-:TechnologyVersion: 2.0
-:ReleaseDate: May 2021
-:CopyrightDates: 2017, 2021
-:TechnologyRI: Eclipse GlassFish 6.1
+:TechnologyVersion: 3.0
+:ReleaseDate: March 2022
```

Comment:
We can update to April

Reply:
I updated it to May instead  as that month is more likely at this point.